editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -522,8 +535,6 @@ Sprite2DEditor::Sprite2DEditor() {
 	add_child(err_dialog);
 
 	debug_uv_dialog = memnew(ConfirmationDialog);
-	debug_uv_dialog->get_ok_button()->set_text(TTR("Create Mesh2D"));
-	debug_uv_dialog->set_title(TTR("Mesh 2D Preview"));
```

Comment:
I removed these lines because they don't achieve anything. The ok button text and title are already set before opening the dialog in `Sprite2DEditor::_menu_option()`.